### DISCUSSION TITLE
Start WebView2 before Windows startup [feat]

### DISCUSSION THREAD

**Discussion by luvor** [tauri-apps/tauri#5919] [General]
### Describe the problem

I was trying run my tauri app instead of default shell (explorer.exe), so I wrote path at userinit.
But, I faced a problem that sometimes app crashes. And sometimes there is shown terminal for 3-5 seconds and then app starts.

Therefore I'm searching for an other solution

### Describe the solution you'd like

To solve this I tried to add to userinit absolute path to EdgeWebView, but the .exe file is inside folder that often changes his name
Like this 108.0.1462.48 => 108.0.1462.54 so I have to rebuild app again.

**Comment by Henrique-CSS** [comment]
Had you tried to put your app path inside "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Run" in windows registry? (Regedit)

**Comment by JonasKruckenberg** [comment]
Have you tried linking to a fixed version of the webview? https://tauri.app/v1/guides/building/windows#fixed-version

I don't necessarily know if that would help, but it's worth a shot anyway I guess. Just know that it comes with very real tradeoffs (bigger size mostly)

**Comment by luvor** [comment]
> Had you tried to put your app path inside "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Run" in windows registry? (Regedit)

Not yet, I will try thanks
But here was an issue with startup cause it's runs after window shell is loaded and when you change default shell it never runs (startup)

Also there is still a problem with finding a version

Recently, I tried to make a script on Rust which finds a path of the version folder, but could not make it because I do not know Rust

**Comment by luvor** [comment]
> Have you tried linking to a fixed version of the webview? https://tauri.app/v1/guides/building/windows#fixed-version
> 
> I don't necessarily know if that would help, but it's worth a shot anyway I guess. Just know that it comes with very real tradeoffs (bigger size mostly)

Yeah, I tried but then met with problems, this software further will be running on many computers so some of them might have WebView already and this code do not work
